src/foam/parsers/HTMLLexer.js

```diff
@@ -219,7 +219,8 @@ foam.CLASS({
             return Embed.create({
               nodeName: v[0][2],
               content: [ v[1] ],
-              attributes: v[0][4]
+              attributes: v[0][4],
+              type: OPEN_CLOSE
             });
```

Comment:
I noticed that `Embed` and `maybeEmbed` did not previously have a type. However, I noticed that the way the grammar is defined, these nodes act like self closing items (i.e., there is only one node for each set of opening and closing tags for these types. Thus, they essentially act as `OPEN_CLOSE`).

Although this change can be discarded, it does make my work a lot easier when performing tag matching.